cli/src/add_url.rs

```diff
@@ -3,7 +3,7 @@ use db::urls::{add_url, AddUrlOutcome, PgPool};
 use uuid::Uuid;
 
 use crate::check_auth_status;
-pub async fn add_url_to_db(
+pub async fn append_url_to_db(
```

Comment:
clippy didn't like the `add_url_to_db` cause the first half `add_url` was named after the function. So I changed it to `append_url_to_db`.

Gotta love clippy😂

Reply:
That definitely works!

I think something thats more in the 'spirit' of the lint could be to just call `to_db` or something since we are already scoped to 'add_url' so we don't really need to say that part again in the function name!
Or what I might do is (similar to what we just did in CLI) call the module `url` and then this can be `add_to_db` or something!

I think the _idea_ behind the clippy lint is that repeating something in the module name and the function name is duplicative, and usually just leads to overly long and verbose names. So instead of trying to use a synonym we can try and remove some of the duplication instead!